props/saber_fett263_buttons.h

```diff
@@ -163,7 +340,13 @@ SaberFett263Buttons() : PropBase() {}
           VolumeUp();
         } else {
           On();
-        }
+#ifdef FETT263_BATTLE_MODE_ALWAYS_ON
+          battle_mode_ = true;
+#endif
+#ifdef FETT263_BATTLE_MODE_START_ON
+          battle_mode_ = true;
+#endif
+	}
```

Comment:
indented weird
However, once we get to a point that the only issues left are indentation, I will just merge it and fix the indentation myself.

Reply:
the indentation looks lined up on my end, I did it in Notepad to avoid issues